UPGRADE.md

```diff
@@ -1,5 +1,20 @@
 # Upgrade to 2.12
 
+## Deprecated colon prefix for prepared statement parameters
+
+The usage of the colon prefix when binding named parameters is deprecated.
+
+    $sql = 'SELECT * FROM users WHERE name = :name OR username = :username';
+    $stmt = $conn->prepare($sql);
+
+    // The usage of the colon is deprecated
+    $stmt->bindValue(':name', $name);
+
+    // Only the parameter name should be passed
+    $stmt->bindValue('username', $username);
+
+    $stmt->execute();
```

Comment:
Let's get some syntactic coloration

```suggestion
```php
$sql = 'SELECT * FROM users WHERE name = :name OR username = :username';
$stmt = $conn->prepare($sql);

// The usage of the colon is deprecated
$stmt->bindValue(':name', $name);

// Only the parameter name should be 
$stmt->bindValue('username', $username);

$stmt->execute();
```

My suggestion does not include the closing backticks, I couldn't figure out how to include them

Reply:
Right, it's markdown. I copy/pasted this block from the documentation which is reStructuredText.